notebooks/reprojection/reproject_demo.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b8f82eaa-bed0-4643-927a-e311918afbd4",
+   "metadata": {},
+   "source": [
+    "# Reprojection Demo\n",
+    "\n",
+    "A notebook on how to use the `kbmod.reprojection` module to reproject WorkUnits into a common WCS frame of reference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5ca16b19-27be-4569-b3f8-644e81604895",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kbmod.work_unit import WorkUnit\n",
+    "from kbmod import reprojection\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from astropy.nddata import CCDData\n",
+    "\n",
+    "import numpy as np\n",
+    "from os import path\n",
+    "\n",
+    "\n",
+    "# visualization utility.\n",
+    "def plot_images(obstimes, image1=None, image2=None, image3=None, image4=None):\n",
+    "    \"\"\"Plot four images side by side\"\"\"\n",
```

Comment:
It might be helpful to mention in the comment that the images are `CCDData` objects. It initially confused me as to how the images had a `wcs` attribute until I got to a lower cell.